h2o-r/h2o-package/R/explain.R

```diff
@@ -2387,6 +2387,349 @@ h2o.ice_plot <- function(model,
   })
 }
 
+#' Learning Curve Plot
+#'
+#' Create learning curve plot for an H2O Model. Learning curves show error metric dependence on
+#' learning progress, e.g., RMSE vs number of trees trained so far in GBM. There can be up to 4 curves
+#' showing Training, Validation, Training on CV Models, and Cross-validation error.
+#'
+#' @param model an H2O model
+#' @param metric Metric to be used for the learning curve plot. These should mostly correspond with stopping metric.
+#' @param cv_ribbon if True, plot the CV mean as a and CV standard deviation as a ribbon around the mean,
+#'                  if NULL, it will attempt to automatically determine if this is suitable visualisation
+#' @param cv_lines if True, plot scoring history for individual CV models, if NULL, it will attempt to
+#'                 automatically determine if this is suitable visualisation
+#'
```

Comment:
New doc string here. Python version is basically the same except for `NULL -> None`